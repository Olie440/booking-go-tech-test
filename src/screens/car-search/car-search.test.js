import React from 'react';
import { shallow } from 'enzyme';
import CarSearchScreen from './car-search.component';

it('renders without crashing', () => {
  const component = shallow(<CarSearchScreen />);
  expect(component).toMatchSnapshot();
});
