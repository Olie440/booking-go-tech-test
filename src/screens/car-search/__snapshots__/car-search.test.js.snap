// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="car-search screen background--blue"
>
  <div
    className="car-search__search-container background--yellow"
  >
    <h2
      className="car-search__header"
    >
      Letâ€™s find your ideal car
    </h2>
    <label
      className="car-search__label"
    >
      Pick-up Location
    </label>
    <input
      className="car-search__input"
      placeholder="city, airport, station, region and district..."
    />
  </div>
</div>
`;
